src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -391,7 +391,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       This condition needs to be inside the target because it the itemgroup will be populated after the target's
       condition is evaluated.
     -->
-    <ItemGroup Condition="'@(RazorGenerate)'!=''">
+    <ItemGroup>
```

Comment:
Will this no op if the file isn't there? This was done on purpose so that we're not generating this extra set of items when you have no Razor files.